# 종합 데이터
std_msgs/Header header
geometry_msgs/Point a_global
float32 a_yaw
v2v_com/ObstacleData[] obstacles